98 Commented/00 Intro/00 Boilerplate/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+/*
+WEBPACK is a module bundler for modern JavaScript applications. 
+When webpack processes your application, it recursively builds a dependency graph,
+that includes every module your application needs, then packages all of those modules into one or more bundles.
+
+There are a lot of statements that have been standardized in ES2015. 
+Although they are not supported in most browsers yet, webpack does support them out of the box.
+
+Webpack "transpiles" the code so that older browsers can also run it.
```

Comment:
Nice intro, but on the last two paragraphs I would rather say:

Webpack integrated well with transpilers like babel or typescript, they allow us...

Webpack itself does not convert from ES6 to ES5 is babel or typescript the ones that do so.

Reply:
Ok, changes read and commit done with #comment1 (this changes has been made on the three projects that I had downloaded)